docs/source/en/model_doc/llama.mdx

```diff
@@ -37,6 +37,8 @@ tokenizer = transformers.LlamaTokenizer.from_pretrained("/output/path/tokenizer/
 model = transformers.LlamaForCausalLM.from_pretrained("/output/path/llama-7b/")
 ```
 
+- The LLaMA tokenizer is based on [sentencepiece](https://github.com/google/sentencepiece). One quick of sentencepiece is that when decoding a sequence, if the first token is the start of the word (e.g. "Banana"), the tokenizer doesn't not prepend the prefix space to the string. To have the tokenizer output the prefix space, set `decode_with_prefix_space=True` in the `LlamaTokenizer` object or in the tokenizer configuration.
```

Comment:
Probably `doesn't not prepend` -> `doesn't prepend` as well.

Reply:
Maybe I missed it, but the `sentencepiece` package should be added to the requirements? Or at least fail with explicit dependency failure rather than not importing the tokenizer.
Unsure it's the right place to comment though, sorry.